### DISCUSSION TITLE
user password reset

### DISCUSSION THREAD

**Discussion by AI-it-is** [fief-dev/fief#243] [Q&A]
Hello, I wish to inform user with password reset link which user can use immediately after user creation by admin.

Here is the use case.
- Admin creates user with random password using fief workspace.
- User gets notified with registration details including password reset link.

I am using webhook event to know about the user creation at my backend, but I am not able to figure out if there is a way to generate the password reset link for the user (valid with defined timeout and single time use).

Is this possible to achieve with current release of fief?
Would appreciate your help.

Regards
Qnal

**Comment by cfredericksen** [comment]
My work around is to create a new user via the Admin API with an embedded password. Once created, a Welcome email will be sent and it informs the user to reset their password.
